cmd/cluster.go

```diff
@@ -1026,7 +1026,7 @@ func (c *config) updateCluster(args []string) error {
 				}
 			}
 			if worker == nil && !removeworkergroup {
-				fmt.Println("Adding a new worker group to the cluster. Please note that running multiple worker groups leads to higher basic costs of the cluster!")
+				fmt.Printf("Adding a new worker group to the cluster:%q. Please note that running multiple worker groups leads to higher basic costs of the cluster!\n", *current.Name)
```

Comment:
I think it would pop out even more if we did this with a color like we did at some other places?

```suggestion
				fmt.Printf("Adding a new worker group to cluster %s. Please note that running multiple worker groups leads to higher basic costs of the cluster!\n", color.GreenString(*current.Name))
```